pycbc/waveform/generator.py

```diff
@@ -306,8 +315,8 @@ def _postgenerate(self, res):
             tapermethod = self.current_params['taper']
             for mode in res:
                 ulm, vlm = res[mode]
-                ulm = taper_timeseries(ulm, tapermethod=tapermethod)
-                vlm = taper_timeseries(vlm, tapermethod=tapermethod)
+                ulm = ulm.taper_timeseries(location=tapermethod)
```

Comment:
We might want to clarify between the method and the location now in the naming. Also, how does this allow using the constant window? It would seem to not allow it.